### PR TITLE
Part of #21421: replace define with constexpr

### PR DESCRIPTION
**Pull request by janclod**
Part of #21421 

Please note that I deleted few `#define` which are never used, only declared. Is this what we want? Do we prefer to keep?

### CODE REVIEW

File: src/openrct2/ride/VehicleRiderControl.cpp [hunk 1]
```diff
@@ -10,7 +10,7 @@
 #include "../entity/EntityRegistry.h"
 #include "Vehicle.h"
 
-constexpr int operator"" _MPH(unsigned long long x)
+constexpr int operator"" _MPH(unsigned long long x) noexcept
```
Comment (Gymnasiast) [inline src/openrct2/ride/VehicleRiderControl.cpp:13]: Could you explain this change?

**Comment by Gymnasiast** [comment]
> Please note that I deleted few #define which are never used, only declared. Is this what we want? 

Removing unused defines is indeed what we want.

File: src/openrct2/ride/VehicleRiderControl.cpp (hunk 1 above)
Comment (janclod) [reply src/openrct2/ride/VehicleRiderControl.cpp:13]: This is something I forgot when refactoring the `_MPH` operator. AFAIK it is good practice to label function as `noexcept` when this is the case, therefore added.

**Comment by janclod** [comment]
Can this be merged? (kind reminder :hugs:)

**Comment by tupaschoal** [comment]
> Can this be merged? (kind reminder :hugs:)

Please wait for at least 1w before pinging, as 1w for merging is already quite fast for the repo

**Comment by janclod** [comment]
> > Can this be merged? (kind reminder 🤗)
> 
> Please wait for at least 1w before pinging, as 1w for merging is already quite fast for the repo

Sorry I did not mean to put pressure... I hope that the 'kind reminder with hugs' would help make it come across as a gentle nudge :) ...and also good to know that I should wait 1w...

**Review by tupaschoal — APPROVED** [review]

**Pull request merged**